ethcore/sync/src/chain/mod.rs

```diff
@@ -155,11 +155,11 @@ const MAX_TRANSACTION_PACKET_SIZE: usize = 8 * 1024 * 1024;
 const MAX_TRANSACTIONS_TO_QUERY: usize = 4096;
 // Maximal number of transactions in sent in single packet.
 const MAX_TRANSACTIONS_TO_PROPAGATE: usize = 64;
-// Min number of blocks to be behind for a snapshot sync
-const SNAPSHOT_RESTORE_THRESHOLD: BlockNumber = 30000;
-const SNAPSHOT_MIN_PEERS: usize = 3;
+/// Min number of blocks to be behind for a snapshot sync
+pub const SNAPSHOT_RESTORE_THRESHOLD: BlockNumber = 30000;
+pub const SNAPSHOT_MIN_PEERS: usize = 3;
 
-const STATUS_PACKET: u8 = 0x00;
+pub const STATUS_PACKET: u8 = 0x00;
```

Comment:
document all the public constants. Also, why have these become public?

Reply:
Because currently, the warp sync handshake packet uses it: https://github.com/paritytech/parity-ethereum/blob/7ad556346eed03cfea2999f7f9daf864de6489af/ethcore/sync/src/chain/mod.rs#L890-L910